package.json

```diff
@@ -31,8 +31,13 @@
   },
   "homepage": "https://github.com/redhat-developer/app-services-ui-components#readme",
   "devDependencies": {
+    "@babel/core": "^7.15.5",
+    "@types/react": "^17.0.24",
+    "@types/react-dom": "^17.0.9",
+    "@types/react-router-dom": "^5.3.0",
     "@typescript-eslint/eslint-plugin": "4.31.2",
     "@typescript-eslint/parser": "4.31.2",
+    "babel-loader": "^8.2.2",
```

Comment:
why do we need babel loader? I thought we are using tsc...

Reply:
Ah yes! It's from Storybook, I had it in this branch and then removed in favor of a smaller PR. SB will use it, we will not for our build